src/interfaces/ICoWSwapOnchainOrders.sol

```diff
@@ -42,4 +42,9 @@ interface ICoWSwapOnchainOrders {
         OnchainSignature signature,
         bytes data
     );
+
+    /// @dev Event emitted to notify that an order was deleted.
+    ///
+    /// @param orderUid CoW Swap's unique order identifier of the order that has been cancelled.
+    event OrderDeletion(bytes indexed orderUid);
```

Comment:
SHould we call it OrderInvalidation?

https://github.com/cowprotocol/services/pull/718#discussion_r1013849572

Reply:
Changed. I like it because it's consistent with the terminology of the settlement contract, I dislike it because it doesn't actually invalidate the order in the contract (it just makes the signature invalid). Overall I think the change is good.
Should we also rename the function `deleteOrder`?

Reply:
Yeah, probably its more consistent to change it there as well. On the other hand we have already audited it.

=> I think we should do it anyways.

Sorry for bringing it up so late.